### ISSUE TITLE
preview images on apps with more than one window.

### ISSUE THREAD

**Issue by drmhessler** [dangvd/crystal-dock#78]
Very nice application, runs smoothly here on Kubuntu 24.10, with Plasma 6.  One missing feature for improvement:
Would it be possible to add preview images for apps with more than one window when hovering about the application icon?

**Comment by dangvd** [comment]
Thank you!
About the feature request, I'll have a look when I can.
Also would feature request #67 be useful for you?

**Comment by drmhessler** [comment]
As I have just seen the bandage points, numbering the windows together with swapping through by left mouse click is sufficient for looking through the windows. Thank you.

**Comment by dangvd** [comment]
Hi,

From the visual indicators under the icon, the user can already see the number of windows and which one is active. Thus I think showing n/size might be redundant?

**Comment by drmhessler** [comment]
Yes, indeed up to 4 windows the indicators are sufficient, and fit well for this purpose. For five and more windows showing the explicit numbering would be i.m.h.o fit better. Any window with number 4 and above gets slightly miss indicated as the last index.

**Comment by dangvd** [comment]
> As small enhancement, one could add the number of the window in the format (n/size):
> 
> label_ + " ("+ (getActiveTask()>=0 ? QString::number(getActiveTask()+1)+"/": "")+ QString::number(tasks_.size()) + ")" : label_;

Spawned #111 for this.
